barretenberg/cpp/src/barretenberg/ultra_honk/decider_verifier.hpp

```diff
@@ -32,7 +32,6 @@ template <typename Flavor> class DeciderVerifier_ {
     bool verify_proof(const DeciderProof&); // used when a decider proof is known explicitly
     bool verify();                          // used when transcript that has been initialized with a proof
     std::shared_ptr<VerificationKey> key;
-    std::map<std::string, Commitment> commitments;
```

Comment:
unrelated to this PR but while debugging something I discovered that this is entirely unused. Instead we use a VerifierCommitments object that is local to the relevant method.